packages/features/bookings/lib/getBookingResponsesSchema.ts

```diff
@@ -98,12 +98,12 @@ function preprocess<T extends z.ZodType>({
         const views = bookingField.views;
         const isFieldApplicableToCurrentView =
           currentView === "ALL_VIEWS" ? true : views ? views.find((view) => view.id === currentView) : true;
+        let hidden = bookingField.hidden;
+        if (bookingField.hideWhenJustOneOption) {
+          hidden = hidden || bookingField.options?.length === 1;
```

Comment:
@hariombalhara  should be <=1 as some event types could have empty array as location.

Reply:
I thought about this. A field with no options shouldn't be allowed to be created ideally. But we can still improve this check like this probably